### ISSUE TITLE
Following documentation/installation guide leads to "Unexpected duplicate task" error

### ISSUE THREAD

**Issue by JoshHrach** [fastlane/fastlane#20486]
### New Issue Checklist

- [X] Updated fastlane to the latest version
- [X] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [X] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [X] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

I'm trying out the Fastlane.swift beta. I'm following the steps as shown in the [documentation](https://docs.fastlane.tools/getting-started/ios/fastlane-swift/). I installed Fastlane with the intent for screenshots. After step 1, the console output suggests trying out my new setup by running `fastlane screenshots`. 

I run this and see the following error: "Unexpected duplicate tasks"

If I open the project as described in step 2 at `[project]/fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj` and try to build it, I also see the same output.

<img width="329" alt="image" src="https://user-images.githubusercontent.com/1069075/179872372-3cbd1a60-c209-4d5c-ab19-dbdcd00fc312.png">

##### Command executed

`fastlane screenshots`

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre> joshhrach@Joshs-MaxBook-Pro DoMarks % fastlane screenshots --capture_output

[✔] 🚀 
[17:52:29]: fastlane detected a Gemfile in the current directory
[17:52:29]: However, it seems like you didn't use `bundle exec`
[17:52:29]: To launch fastlane faster, please use
[17:52:29]: 
[17:52:29]: $ bundle exec fastlane screenshots --capture_output
[17:52:29]: 
[17:52:29]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
DEBUG [2022-07-19 17:52:29.81]: Checking if there are any plugins that should be loaded...
INFO [2022-07-19 17:52:29.93]: loading manifest: /opt/homebrew/Cellar/fastlane/2.208.0/libexec/gems/fastlane-2.208.0/fastlane/swift/upgrade_manifest.json
DEBUG [2022-07-19 17:52:29.96]: No missing groups found, so we don't need to worry about adding new groups
DEBUG [2022-07-19 17:52:29.96]: Actions.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.56/0.9.56
DEBUG [2022-07-19 17:52:29.96]: Fastlane.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.160/0.9.160
DEBUG [2022-07-19 17:52:29.96]: DeliverfileProtocol.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.107/0.9.107
DEBUG [2022-07-19 17:52:29.96]: GymfileProtocol.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.110/0.9.110
DEBUG [2022-07-19 17:52:29.96]: MatchfileProtocol.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.104/0.9.104
DEBUG [2022-07-19 17:52:29.96]: Plugins.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.56/0.9.56
DEBUG [2022-07-19 17:52:29.96]: PrecheckfileProtocol.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.103/0.9.103
DEBUG [2022-07-19 17:52:29.96]: ScanfileProtocol.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.115/0.9.115
DEBUG [2022-07-19 17:52:29.96]: ScreengrabfileProtocol.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.105/0.9.105
DEBUG [2022-07-19 17:52:29.96]: SnapshotfileProtocol.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.99/0.9.99
DEBUG [2022-07-19 17:52:29.96]: LaneFileProtocol.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.2/0.9.2
DEBUG [2022-07-19 17:52:29.96]: ControlCommand.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.2/0.9.2
DEBUG [2022-07-19 17:52:29.96]: RubyCommand.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.2/0.9.2
DEBUG [2022-07-19 17:52:29.96]: RubyCommandable.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.2/0.9.2
DEBUG [2022-07-19 17:52:29.96]: Runner.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.2/0.9.2
DEBUG [2022-07-19 17:52:29.96]: SocketClient.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.2/0.9.2
DEBUG [2022-07-19 17:52:29.96]: SocketClientDelegateProtocol.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.2/0.9.2
DEBUG [2022-07-19 17:52:29.96]: SocketResponse.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.2/0.9.2
DEBUG [2022-07-19 17:52:29.96]: main.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.2/0.9.2
DEBUG [2022-07-19 17:52:29.96]: ArgumentProcessor.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.2/0.9.2
DEBUG [2022-07-19 17:52:29.96]: RunnerArgument.swift FastlaneRunnerAPIVersion (bundled/target): 0.9.2/0.9.2
DEBUG [2022-07-19 17:52:29.96]: Checking for new user-provided tool configuration files
INFO [2022-07-19 17:52:29.96]: FastlaneSwiftRunner project is up-to-date
DEBUG [2022-07-19 17:52:29.96]: No runner found, setting re-build runner flag
DEBUG [2022-07-19 17:52:29.96]: Building FastlaneSwiftRunner
WARN [2022-07-19 17:52:29.98]: Resolving Swift Package Manager dependencies...
INFO [2022-07-19 17:52:29.98]: $ xcodebuild -resolvePackageDependencies -scheme FastlaneRunner -project ./fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj
INFO [2022-07-19 17:52:30.20]: ▸ 2022-07-19 17:52:30.206 xcodebuild[89980:1723558] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
INFO [2022-07-19 17:52:30.20]: ▸ 2022-07-19 17:52:30.207 xcodebuild[89980:1723558] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
INFO [2022-07-19 17:52:30.32]: ▸ Command line invocation:
INFO [2022-07-19 17:52:30.32]: ▸     /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -resolvePackageDependencies -scheme FastlaneRunner -project ./fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj
INFO [2022-07-19 17:52:30.32]: ▸ User defaults from command line:
INFO [2022-07-19 17:52:30.32]: ▸     IDEPackageSupportUseBuiltinSCM = YES
INFO [2022-07-19 17:52:30.60]: ▸ --- xcodebuild: WARNING: Using the first of multiple matching destinations:
INFO [2022-07-19 17:52:30.60]: ▸ { platform:macOS, arch:arm64, id:00006001-001031E90140011E }
INFO [2022-07-19 17:52:30.60]: ▸ { platform:macOS, arch:x86_64, id:00006001-001031E90140011E }
INFO [2022-07-19 17:52:30.60]: ▸ { platform:macOS, name:Any Mac }
INFO [2022-07-19 17:52:30.60]: ▸ resolved source packages: 
INFO [2022-07-19 17:52:30.60]: $ xcodebuild -showBuildSettings -scheme FastlaneRunner -project ./fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj
2022-07-19 17:52:30.810 xcodebuild[89986:1723636] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
2022-07-19 17:52:30.810 xcodebuild[89986:1723636] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
--- xcodebuild: WARNING: Using the first of multiple matching destinations:
{ platform:macOS, arch:arm64, id:00006001-001031E90140011E }
{ platform:macOS, arch:x86_64, id:00006001-001031E90140011E }
{ platform:macOS, name:Any Mac }
DEBUG [2022-07-19 17:52:31.28]: Parsing project file '/Users/joshhrach/Work/Apps/Repos/DoMarks/fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj' to find selected provisioning profiles
DEBUG [2022-07-19 17:52:31.28]: Finding provision profiles for 'Release'
DEBUG [2022-07-19 17:52:31.29]: Merging provisioning profile mappings
DEBUG [2022-07-19 17:52:31.29]: -------------------------------------
DEBUG [2022-07-19 17:52:31.29]: Primary provisioning profile mapping:
DEBUG [2022-07-19 17:52:31.29]: {}
DEBUG [2022-07-19 17:52:31.29]: Secondary provisioning profile mapping:
DEBUG [2022-07-19 17:52:31.29]: {}
DEBUG [2022-07-19 17:52:31.29]: Resulting in the following mapping:
DEBUG [2022-07-19 17:52:31.29]: {}
INFO [2022-07-19 17:52:31.33]: $ set -o pipefail && xcodebuild -scheme FastlaneRunner -project ./fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj -destination 'generic/platform=macOS' build | tee /Users/joshhrach/Library/Logs/gym/App-FastlaneRunner.log | xcpretty
INFO [2022-07-19 17:52:31.54]: ▸ 2022-07-19 17:52:31.540 xcodebuild[90000:1723728] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
INFO [2022-07-19 17:52:31.54]: ▸ 2022-07-19 17:52:31.540 xcodebuild[90000:1723728] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
INFO [2022-07-19 17:52:32.03]: ▸ ❌  error: Unexpected duplicate tasks
INFO [2022-07-19 17:52:32.03]: ▸ ❌  error: Unexpected duplicate tasks
INFO [2022-07-19 17:52:32.03]: ▸ ** BUILD FAILED **
ERROR [2022-07-19 17:52:32.03]: Exit status: 65
WARN [2022-07-19 17:52:32.03]: Lane Context:
INFO [2022-07-19 17:52:32.03]: {:LANE_NAME=>"screenshots"}
ERROR [2022-07-19 17:52:32.03]: Exit status: 65
ERROR [2022-07-19 17:52:32.03]: fastlane finished with errors </pre>
</details>

**Comment by JoshHrach** [comment]
Per the template.
> Please run `fastlane env` and copy the output below. This will help us help you.
If you used the `--capture_output` option, please remove this block as it is already included there. 

However, as the bot is asking for it:

<details>
  <pre> <details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                      |
| --------------------------- | ---------------------------------------------------------- |
| OS                          | 12.4                                                       |
| Ruby                        | 3.1.2                                                      |
| Bundler?                    | false                                                      |
| Git                         | git version 2.32.1 (Apple Git-133)                         |
| Installation Source         | /opt/homebrew/Cellar/fastlane/2.208.0/libexec/bin/fastlane |
| Host                        | macOS 12.4 (21F79)                                         |
| Ruby Lib Dir                | /opt/homebrew/Cellar/ruby/3.1.2/lib                        |
| OpenSSL Version             | OpenSSL 1.1.1n  15 Mar 2022                                |
| Is contained                | false                                                      |
| Is homebrew                 | true                                                       |
| Is installed via Fabric.app | false                                                      |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                |
| Xcode Version               | 13.4                                                       |
| Swift Version               | 5.6.1                                                      |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |


### fastlane files:

<details><summary>`./fastlane/Fastfile.swift`</summary>

```ruby
// This file contains the fastlane.tools configuration
// You can find the documentation at https://docs.fastlane.tools
//
// For a list of all available actions, check out
//
//     https://docs.fastlane.tools/actions
//

import Foundation

class Fastfile: LaneFile {
	func screenshotsLane() {
	desc("Generate new localized screenshots")
		captureScreenshots(scheme: "MyApp (Marketing iOS)")
	}
}

```
</details>

**No Appfile found**


### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.208.0 | ✅ Up-To-Date  |


### Loaded fastlane plugins:

**No plugins Loaded**


<details><summary><b>Loaded gems</b></summary>

| Gem                             | Version      |
| ------------------------------- | ------------ |
| error_highlight                 | 0.3.0        |
| did_you_mean                    | 1.6.1        |
| atomos                          | 0.1.3        |
| rexml                           | 3.2.5        |
| CFPropertyList                  | 3.0.5        |
| claide                          | 1.1.0        |
| colored2                        | 3.1.2        |
| nanaimo                         | 0.3.0        |
| xcodeproj                       | 1.22.0       |
| rouge                           | 2.0.7        |
| xcpretty                        | 0.3.0        |
| terminal-notifier               | 2.0.0        |
| unicode-display_width           | 1.8.0        |
| terminal-table                  | 1.8.0        |
| plist                           | 3.6.0        |
| public_suffix                   | 4.0.7        |
| addressable                     | 2.8.0        |
| multipart-post                  | 2.0.0        |
| word_wrap                       | 1.0.0        |
| optparse                        | 0.1.1        |
| tty-screen                      | 0.8.1        |
| tty-cursor                      | 0.7.1        |
| tty-spinner                     | 0.9.3        |
| artifactory                     | 3.0.15       |
| babosa                          | 1.0.4        |
| colored                         | 1.2          |
| highline                        | 2.0.3        |
| commander                       | 4.6.0        |
| excon                           | 0.92.3       |
| faraday-em_http                 | 1.0.0        |
| faraday-em_synchrony            | 1.0.0        |
| faraday-excon                   | 1.1.0        |
| faraday-httpclient              | 1.0.1        |
| faraday-multipart               | 1.0.4        |
| faraday-net_http                | 1.0.1        |
| faraday-net_http_persistent     | 1.2.0        |
| faraday-patron                  | 1.0.0        |
| faraday-rack                    | 1.0.0        |
| faraday-retry                   | 1.0.3        |
| ruby2_keywords                  | 0.0.5        |
| faraday                         | 1.10.0       |
| unf_ext                         | 0.0.8.2      |
| unf                             | 0.1.4        |
| domain_name                     | 0.5.20190701 |
| http-cookie                     | 1.0.5        |
| faraday-cookie_jar              | 0.0.7        |
| faraday_middleware              | 1.2.0        |
| fastimage                       | 2.2.6        |
| gh_inspector                    | 1.1.3        |
| mini_magick                     | 4.11.0       |
| naturally                       | 2.2.1        |
| rubyzip                         | 2.3.2        |
| security                        | 0.1.3        |
| xcpretty-travis-formatter       | 1.0.1        |
| dotenv                          | 2.7.6        |
| bundler                         | 2.3.11       |
| simctl                          | 1.6.8        |
| jwt                             | 2.4.1        |
| uber                            | 0.1.0        |
| declarative                     | 0.0.20       |
| trailblazer-option              | 0.1.2        |
| representable                   | 3.2.0        |
| retriable                       | 3.1.2        |
| mini_mime                       | 1.1.2        |
| memoist                         | 0.16.2       |
| multi_json                      | 1.15.0       |
| os                              | 1.1.4        |
| signet                          | 0.17.0       |
| googleauth                      | 1.2.0        |
| httpclient                      | 2.8.3        |
| webrick                         | 1.7.0        |
| google-apis-core                | 0.7.0        |
| google-apis-playcustomapp_v1    | 0.10.0       |
| google-apis-androidpublisher_v3 | 0.25.0       |
| google-cloud-env                | 1.6.0        |
| google-cloud-errors             | 1.2.0        |
| google-cloud-core               | 1.6.0        |
| google-apis-iamcredentials_v1   | 0.13.0       |
| google-apis-storage_v1          | 0.18.0       |
| rake                            | 13.0.6       |
| digest-crc                      | 0.6.4        |
| google-cloud-storage            | 1.37.0       |
| emoji_regex                     | 3.2.3        |
| jmespath                        | 1.6.1        |
| aws-partitions                  | 1.608.0      |
| aws-eventstream                 | 1.2.0        |
| aws-sdk-core                    | 3.131.3      |
| aws-sdk-kms                     | 1.58.0       |
| aws-sdk-s3                      | 1.114.0      |
| set                             | 1.0.2        |
| json                            | 2.6.2        |
| aws-sigv4                       | 1.5.1        |
| forwardable                     | 1.3.2        |
| logger                          | 1.5.0        |
| pathname                        | 0.2.0        |
| shellwords                      | 0.1.0        |
| cgi                             | 0.3.1        |
| date                            | 3.2.2        |
| timeout                         | 0.2.0        |
| stringio                        | 3.0.1        |
| securerandom                    | 0.1.1        |
| uri                             | 0.11.0       |
| openssl                         | 3.0.0        |
| digest                          | 3.1.0        |
| io-nonblock                     | 0.1.0        |
| ipaddr                          | 1.2.4        |
| io-wait                         | 0.2.1        |
| zlib                            | 2.1.1        |
| resolv                          | 0.2.1        |
| time                            | 0.2.0        |
| open-uri                        | 0.2.0        |
| mutex_m                         | 0.1.1        |
| net-protocol                    | 0.1.2        |
| net-http                        | 0.2.0        |
| ostruct                         | 0.5.2        |
| english                         | 0.7.1        |
| erb                             | 2.2.3        |
| strscan                         | 3.0.1        |
| abbrev                          | 0.1.0        |
| io-console                      | 0.5.11       |
| tempfile                        | 0.1.2        |
| delegate                        | 0.2.0        |
| fileutils                       | 1.6.0        |
| tmpdir                          | 0.1.2        |
| base64                          | 0.1.1        |
| singleton                       | 0.1.1        |
| open3                           | 0.1.1        |
| nkf                             | 0.1.1        |
| prettyprint                     | 0.1.1        |
| pp                              | 0.3.0        |
| find                            | 0.1.1        |
| yaml                            | 0.2.0        |
| psych                           | 4.0.3        |
</details>


*generated on:* **2022-07-19**
</details> </pre>
</details>
